subscription-service/src/main/resources/application.properties

```diff
@@ -30,7 +30,12 @@ application.authentication.enabled = false
 # application.graylog.host = localhost
 # application.graylog.port = 12201
 # application.graylog.source = stellio-int
-
 # Pagination config for query resources endpoints
-application.pagination.limit-default = 30
-application.pagination.limit-max = 100
+application.pagination.limit-default=30
+application.pagination.limit-max=100
+#configure ressource for authentification
+spring.security.oauth2.client.registration.keycloak.authorization-grant-type=client_credentials
+spring.security.oauth2.client.registration.keycloak.client-id=senseen-gael
+spring.security.oauth2.client.registration.keycloak.client-secret=221d117c-b7fd-4b8d-8064-f9c51605c81c
+spring.security.oauth2.client.provider.keycloak.token-uri=https://sso.eglobalmark.com/auth/realms/stellio-dev/protocol/openid-connect/token
+spring.security.oauth2.client.provider.keycloak.base-url=https://stellio-dev.eglobalmark.com
```

Comment:
this is not related to KC (check how similar property is named in search service)

Reply:
I change this